frontend/src/static/js/v_summary.js

```diff
@@ -828,15 +827,6 @@ window.vSummary = {
         }
       });
 
-      const filterControl = {
```

Comment:
This also exists around line number 300. Do you know why did we have duplicate code?

Reply:
The one in 300 is needed as summary_charts will receive the filtered repo and display them without sorting. The line here is for zoom, which will perform sorting in its own js file.